### ISSUE TITLE
generated associated migrations miss ActiveRecord::Migration version number

### ISSUE THREAD

**Issue by dfherr** [airblade/paper_trail#971]
title says it all. I guess this is simpler for you and me without your bug_report template.
running:

```
rails generate paper_trail:install --with-associations
```

will generate association migrations without the Rails version, as required by rails 5.x

```
class CreateVersionAssociations < ActiveRecord::Migration
```

the regular createVersions migrations is fine:

```
class CreateVersions < ActiveRecord::Migration[5.1]
```

**Comment by jaredbeck** [comment]
This was fixed by #949 in release 7.0.1. Closing, but please let us know if upgrading to > 7.0.1 does not fix this for you.

**Comment by jaredbeck** [comment]
Dennis, regarding your comment in #949 ..

> i was actually running 7.0.2. At least this is what my gemfile lock says:
>   paper_trail (7.0.2)
>    activerecord (>= 4.0, < 5.2)
>    request_store (~> 1.1)

I tried running the generator in a brand new rails 5.1 app with PT 7.0.2 and it worked for me.

Maybe you forgot to use `bundle exec`, or maybe you're using a different version of rails?

What steps will reproduce this issue? Thanks.

**Comment by dfherr** [comment]
Ok i could reproduce this with a new rails 5.1.1 project using a postgres db:


steps:

1. create new rails project (I get an error here when the setup runs bundle install, because I do not have sqllite installed and installing it fails. Rails runs flawlessly though, after I change the Gemfile to use pg, so paper_trail shouldn't be influenced by this)
2. change Gemfile to minimal Gemfile using postgres (see below)
3. setup database.yml to be able to connect to pg
4. run `bundle exec rails generate paper_trail:install --with-associations`

My "minimal" Gemfile (this is the default file generated by rails, plus paper_trail and pg, minus a lot of irrelevant stuff):
```
source 'https://rubygems.org'

git_source(:github) do |repo_name|
  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
  "https://github.com/#{repo_name}.git"
end

gem 'paper_trail'
# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '~> 5.1.1'
# Use Puma as the app server
gem 'puma', '~> 3.7'
gem "pg"


group :development do
  gem 'listen', '>= 3.0.5', '< 3.2'
  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
  gem 'web-console', '>= 3.3.0'
  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
end
```

my gemfile lock:

```
GEM
  remote: https://rubygems.org/
  specs:
    actioncable (5.1.1)
      actionpack (= 5.1.1)
      nio4r (~> 2.0)
      websocket-driver (~> 0.6.1)
    actionmailer (5.1.1)
      actionpack (= 5.1.1)
      actionview (= 5.1.1)
      activejob (= 5.1.1)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 2.0)
    actionpack (5.1.1)
      actionview (= 5.1.1)
      activesupport (= 5.1.1)
      rack (~> 2.0)
      rack-test (~> 0.6.3)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (5.1.1)
      activesupport (= 5.1.1)
      builder (~> 3.1)
      erubi (~> 1.4)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.3)
    activejob (5.1.1)
      activesupport (= 5.1.1)
      globalid (>= 0.3.6)
    activemodel (5.1.1)
      activesupport (= 5.1.1)
    activerecord (5.1.1)
      activemodel (= 5.1.1)
      activesupport (= 5.1.1)
      arel (~> 8.0)
    activesupport (5.1.1)
      concurrent-ruby (~> 1.0, >= 1.0.2)
      i18n (~> 0.7)
      minitest (~> 5.1)
      tzinfo (~> 1.1)
    arel (8.0.0)
    bindex (0.5.0)
    builder (3.2.3)
    concurrent-ruby (1.0.5)
    erubi (1.6.0)
    ffi (1.9.18)
    globalid (0.4.0)
      activesupport (>= 4.2.0)
    i18n (0.8.4)
    listen (3.1.5)
      rb-fsevent (~> 0.9, >= 0.9.4)
      rb-inotify (~> 0.9, >= 0.9.7)
      ruby_dep (~> 1.2)
    loofah (2.0.3)
      nokogiri (>= 1.5.9)
    mail (2.6.5)
      mime-types (>= 1.16, < 4)
    method_source (0.8.2)
    mime-types (3.1)
      mime-types-data (~> 3.2015)
    mime-types-data (3.2016.0521)
    mini_portile2 (2.1.0)
    minitest (5.10.2)
    nio4r (2.1.0)
    nokogiri (1.7.2)
      mini_portile2 (~> 2.1.0)
    paper_trail (7.0.2)
      activerecord (>= 4.0, < 5.2)
      request_store (~> 1.1)
    pg (0.20.0)
    puma (3.9.0)
    rack (2.0.3)
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (5.1.1)
      actioncable (= 5.1.1)
      actionmailer (= 5.1.1)
      actionpack (= 5.1.1)
      actionview (= 5.1.1)
      activejob (= 5.1.1)
      activemodel (= 5.1.1)
      activerecord (= 5.1.1)
      activesupport (= 5.1.1)
      bundler (>= 1.3.0, < 2.0)
      railties (= 5.1.1)
      sprockets-rails (>= 2.0.0)
    rails-dom-testing (2.0.3)
      activesupport (>= 4.2.0)
      nokogiri (>= 1.6)
    rails-html-sanitizer (1.0.3)
      loofah (~> 2.0)
    railties (5.1.1)
      actionpack (= 5.1.1)
      activesupport (= 5.1.1)
      method_source
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (12.0.0)
    rb-fsevent (0.9.8)
    rb-inotify (0.9.8)
      ffi (>= 0.5.0)
    request_store (1.3.2)
    ruby_dep (1.5.0)
    spring (2.0.2)
      activesupport (>= 4.2)
    spring-watcher-listen (2.0.1)
      listen (>= 2.7, < 4.0)
      spring (>= 1.2, < 3.0)
    sprockets (3.7.1)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.2.0)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    thor (0.19.4)
    thread_safe (0.3.6)
    tzinfo (1.2.3)
      thread_safe (~> 0.1)
    web-console (3.5.1)
      actionview (>= 5.0)
      activemodel (>= 5.0)
      bindex (>= 0.4.0)
      railties (>= 5.0)
    websocket-driver (0.6.5)
      websocket-extensions (>= 0.1.0)
    websocket-extensions (0.1.2)

PLATFORMS
  ruby

DEPENDENCIES
  listen (>= 3.0.5, < 3.2)
  paper_trail
  pg
  puma (~> 3.7)
  rails (~> 5.1.1)
  spring
  spring-watcher-listen (~> 2.0.0)
  web-console (>= 3.3.0)

BUNDLED WITH
   1.14.6
```

**Comment by dfherr** [comment]
An additional note:

I could reproduce this using my elder rails project running inside a ubuntu 16.04 docker container using ruby 2.4.1 and with a new rails project on my local installation ubuntu 16.04 with rbenv running ruby 2.2.6p396. postgres might be the issue?

**Comment by jaredbeck** [comment]
Thanks Dennis, I was able to reproduce this using `--with-associations`.

Both `create_version_associations.rb` and `add_transaction_id_column_to_versions.rb` are missing the migration version.

It looks like this was actually fixed by #959, which has not been released yet. I'll release 7.0.3.
